src/hotspot/os/windows/sharedRuntimeRem.cpp

```diff
@@ -25,7 +25,7 @@
 #include "precompiled.hpp"
 #include "runtime/sharedRuntime.hpp"
 
-#ifdef _M_ARM64
+#ifdef AARCH64
```

Comment:
Shouldn't this also be
```
#if defined(_WINDOWS) && defined(AARCH64)
```
?

Reply:
sharedRuntimeRem.cpp is a Windows only file \- Checking for \_WINDOWS for a file under os/windows is somewhat redundant

Reply:
Sorry, missed that. Please ignore.